### PR TITLE
Enable tests and resolve several issues identified by SonarCloud

### LINKED ISSUE #136: docs link on front mage leads to 404
Somehow the "docs" link in the README.adoc is dead and leads to 404.

### PR DESCRIPTION
**Pull request by MikeEdgar**
* Remove docs link from README (Fixes #136)
* Reformatted JaxRsTest based on Maven build (change of line endings from Windows to Unix)

### CODE REVIEW

**Comment by MikeEdgar** [comment]
Added JaCoCo plugin as well

File: pom.xml [hunk 1]
```diff
@@ -191,7 +191,26 @@
                 <plugin>
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>build-helper-maven-plugin</artifactId>
-                    <version>{version.buildhelper.plugin}</version>
```
Comment (EricWittmann) [inline pom.xml]: In the category of "how did this ever work"??  :)

**Comment by MikeEdgar** [comment]
The clean/verify/sonar step will also need to be updated to run with the "coverage" Maven profile.

File: pom.xml [hunk 2]
```diff
@@ -191,7 +191,26 @@
...
+                    <version>${version.buildhelper.plugin}</version>
+                </plugin>
+                <plugin>
```
Comment (kenfinnigan) [inline pom.xml]: This should be in the parent right? Not needed here

Comment (MikeEdgar) [reply pom.xml]: @kenfinnigan - the version is managed in the open-api parent and used in the TCK. This plugin is not present in the smallrye-parent POM at all.

Comment (kenfinnigan) [reply pom.xml]: Should've been specific, I meant the jacoco plugin defined below

Comment (kenfinnigan) [reply pom.xml]: Ugh, after checking, it's not actually defined in SmallRye Parent yet.

Would you be able to submit a PR for that? https://github.com/smallrye/smallrye-parent/issues/19

Comment (MikeEdgar) [reply pom.xml]: Yes, I can do that.

Comment (MikeEdgar) [reply pom.xml]: Opened smallrye/smallrye-parent#21 and removed the plugin configuration from the parent of this project.

Comment (kenfinnigan) [reply pom.xml]: Just released the parent.

In about an hour it should be in Central so you can update PR to use `8`

Comment (MikeEdgar) [reply pom.xml]: Updated to use smallrye-parent version 8.

**Pull request merged**